src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/ReadAndUpdateOperation.java

```diff
@@ -41,85 +43,85 @@ public record ReadAndUpdateOperation(
     boolean returnUpdatedDocument,
     boolean upsert,
     Shredder shredder,
-    int updateLimit)
+    int updateLimit,
+    int retryLimit)
```

Comment:
Although Javadoc params are empty, probably best to add entry for `retryLimit` -- and maybe mention if it's specifically for LWT retries? (or reference to class that has javadoc for that, if any)

Reply:
Added to javadoc